ndk/src/relay/connectivity.ts

```diff
@@ -82,11 +81,14 @@ export class NDKRelayConnectivity {
                 this._status = NDKRelayStatus.CONNECTING;
             else this._status = NDKRelayStatus.RECONNECTING;
 
-            this.relay.off("connect", connectHandler);
-            this.relay.off("disconnect", disconnectHandler);
-            this.relay.on("connect", connectHandler);
-            this.relay.on("disconnect", disconnectHandler);
-            this.relay.on("auth", authHandler);
+            // ===========================
```

Comment:
This is the main remaining thing that needs fixing before testing this more thoroughly to make sure it works.